src/main/scala/cromwell/Main.scala

```diff
@@ -66,6 +67,10 @@ object Main extends App {
     argCapitalized = arg.capitalize
     action <- Actions.values find (_.toString == argCapitalized)
   } yield action
+
+  def deprecationNotice(): Unit = {
+    Console.err.println("This functionality is deprecated. Please use wdltool: https://github.com/broadinstitute/wdltool")
```

Comment:
Also, Cromwell and wdltool appear to have different versioning structures.

Since it's a sine qua non that the versions match up - this message should point to a specific wdltool version. And wdltool should probably have a switch to say which cromwells it supports?